docs/network-binding-plugin.md

```diff
@@ -466,3 +466,28 @@ There may be several options to implement this, one relatively simple
 method could be to preserve the data on a `dummy` interface.
 Aa a network binding plugin author, both the CNI plugin and the sidecar codebase
 is available, therefore both can be in sync to share such information.
+
+## Migration Support
+
+The VM migration operation is considered a core feature in Kubevirt and
+therefore the ability to support it needs to be addressed when designing
+a network binding plugin.
+
+The following points should be addressed in relation to migration support:
+- Kubevirt will allow migration only if the primary network supports it.
+- For a network binding plugin to support migration, the `migration` `method`
+  must be specified in the Kubevirt CR.
+  See the user-guide network binding plugin
+  [migration](https://kubevirt.io/user-guide/virtual_machines/network_binding_plugins/#migration)
+  section on how to define it.
+- The backend `libvirt` and `qemu` needs to support migration for the specific
+  device that is used in the domain.
+  - The `hostdevice` devices in the domain do not support migration out-of-the-box.
+    To support migration with these devices a special handling is required by
+    Kubevirt. The special handling is currently implemented for core SR-IOV
+    binding by unplugging the device at the source before migration is started
+    and plugging it back at the destination after the migration completed.
```

Comment:
This will be incorrect.
We do not have support for `hostdevice` devices through the plugin infrastructure.
The `SR-IOV` binding is not an example of a plugin binding.

I used it to express what is needed to support something like this and explicitly mentioned we do not yet have support for it in the plugins.

Reply:
> The current state is not clear enough, one might think the unplug/plug can be used for any binding plugin

I do not understand how hotplug got involved here.
The text here is about migration and in the specific section about the limitations hostdevices have.

It explicitly says hostdevices are not supported at the moment.

> I suggested to refer to how kubevirt core sr-iov binding does the "special handling" for migration(the uplug/plug thing), not to say its an example for a plugin or plugin support host devices.

It explains how hostdevices are treated by the SR-IOV binding to express the challenge.

I see no point in ref to that binding because it says nothing on how it works internally. That is a user-guide document, it does not enter into such details.

Maybe you should point me to the sentence that is problematic in your opinion here and how one can interpret it wrongly.

Reply:
> I do not understand how hotplug got involved here. The text here is about migration and in the specific section about the limitations hostdevices have.
> 

The unplug/plug core SR-IOV binding does on migration, this PR also mentions.

> Maybe you should point me to the sentence that is problematic in your opinion here and how one can interpret it wrongly.

Its about the phrasing, to me the "The special handling is currently implemented.." past might sound as it can be used for binding plugin.

Anyhow, I am not insisting on it.

Reply:
I rephrased, please take another look.